modules/nw-scaling-ingress-controller.adoc

```diff
@@ -5,33 +5,35 @@
 [id="nw-ingress-controller-configuration_{context}"]
 = Scaling an Ingress Controller
 
-An Ingress Controller can be manually scaled up or down based on
-requirements such as the requirement to increase throughput. `oc`
-commands are used to scale the `IngressController` resource.
-The following procedure provides an example for scaling up the
-default `IngressController`.
+An Ingress Controller can be manually scaled up or down based on routing
```

Comment:
I'm not sure which keywords you're trying to load, but I might start with "Manually scale an Ingress Controller to meet routing performance or availability requirements..."